tensorflow/lite/kernels/pooling_test.cc

```diff
@@ -26,8 +26,22 @@ using ::testing::ElementsAreArray;
 
 class BasePoolingOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, bias, padding types,
-  // stride values.
+  BasePoolingOpModel(const BuiltinOperator& type, const TensorData& input,
+                     const int& filter_width, const int& filter_height,
```

Comment:
It's fine to use: int filter_width, int filter_height instead.

Reply:
@haozha111 Sure, made the change as per the comments, kindly check and approve